cmd/kn/main.go

```diff
@@ -15,14 +15,18 @@
 package main
 
 import (
-	"errors"
+	"context"
 	"fmt"
 	"math/rand"
 	"os"
+	"os/signal"
 	"regexp"
 	"strings"
+	"syscall"
 	"time"
 
+	"github.com/pkg/errors"
```

Comment:
Could the error check be done without adding a new dependency?
```
if errors.Is(err, context.Canceled) {
```

Reply:
Probably yes. Initially I used `error.Cause() == ?` which is not in the standard `errors` package. But  the `Is()` function is part of the standard package (probably since go 1.13).